src/quarto-cli/NOTES.md

```diff
@@ -76,7 +76,7 @@ See also: <https://pptr.dev/troubleshooting#running-puppeteer-in-docker>
 `"latest"` and `"prerelease"` refer to the release and pre-release versions
 listed in [the Quarto CLI's download page](https://quarto.org/docs/download).
 
-You can also specify a version number, like `"1"`, `"1.2"`, `"1.0.38"`.
+You can also specify a version number **greater than** `"1.2"`, like `"1.3"` or `"1.6.42"`.
```

Comment:
Is this correct?
I assumed that if the `quarto install` command was not invoked, no error would occur. Please check the test.

Reply:
Sorry, but there doesn't seem to be any test at the moment that confirms that something like `1.0.38` can be installed.
IIUC, it should still be possible to specify a version like `1.0.38` after this change (if it becomes impossible, I think a major version change of this Feature is needed).